### PR TITLE
Add instructions to replicate prod database to a local docker volume

### PR DESCRIPTION
**Pull request by diversemix**

### CODE REVIEW

File: docs/dkr-copy-volume.md [hunk 1]
```diff
@@ -12,6 +12,23 @@ Assuming you've already copied a volume which had production data in to `prod-da
   dk-copy-volume.sh prod-database navigator_db-data-backend
 ```
 
+## Creating a local volume from prod
```
Comment (THOR300) [inline docs/dkr-copy-volume.md:15]: Are we absolutely sure we want to port forward through to the bastion to the prod database and then do a dump to local? 
Would it not be better to do this against the staging environment?

File: docs/dkr-copy-volume.md [hunk 2]
```diff
@@ -12,6 +12,23 @@ Assuming you've already copied a volume which had production data in to `prod-da
...
+
+- Use `nav-stack-connect.sh` to connect to prod
```
Comment (THOR300) [inline docs/dkr-copy-volume.md:17]: Potentially put the actual commands for these first three lines? 
Maybe reference a document detailing how to login to the bastion?

**Review by THOR300 — COMMENTED** [review]
Just a comment query about whether we want to run this against production and not staging.

I don't whether the documentation for these scripts needs to be as tight as the run book for example but just put a couple comments if so requesting for the actual commands.

File: docs/dkr-copy-volume.md (hunk 1 above)
Comment (diversemix) [reply docs/dkr-copy-volume.md:15]: Can be done with either

File: docs/dkr-copy-volume.md (hunk 2 above)
Comment (diversemix) [reply docs/dkr-copy-volume.md:17]: Not necessary - this script does it all for you

File: docs/dkr-copy-volume.md (hunk 1 above)
Comment (diversemix) [reply docs/dkr-copy-volume.md:15]: At the minute we cannot rely that staging will have production data on there.

**Review by joel-wright — APPROVED** [review]

**Pull request merged**